Data-Science/README.md

```diff
@@ -0,0 +1,176 @@
+- [Power BI complete course -Coursera(Keith Gibson)](https://www.coursera.org/learn/data-driven-decisions-with-power-bi)
+- [Power BI documented tutorial -GFG(geeksforgeeks)](https://www.geeksforgeeks.org/power-bi-tools-and-functionalities/)
+
+### Tableau
+
+- [Tableau data visualisation tutorial -datacamp(Parul Pandey)](https://www.datacamp.com/tutorial/data-visualisation-tableau)
+- [Tableau training -Salesforce](https://www.tableau.com/learn/training)
+- [Tableau Introduction -datacamp](https://learn.datacamp.com/courses/introduction-to-tableau)
+- [Tableau full course -Youtube(simplilearn)](https://youtu.be/HM81vShSlWY)
+- [Tableau documented tutorial -GFG(geeksforgeeks)](<(https://www.geeksforgeeks.org/what-is-tableau-and-its-importance-in-data-visualization/)>)
+
+### Web Scraping/APIs
+
+- [Web Scraping in Python -datacamp](https://learn.datacamp.com/courses/web-scraping-with-python)
+- [Build Web Scraper with python -RealPython(Martin Breuss)](https://realpython.com/beautiful-soup-web-scraper-python/)
+- [Web Scraping with python -Youtube(freeCodeCamp.org)](https://youtu.be/XVv6mJpFOb0)
+
+### Deep Learning
+
+- [Deep Learning with tenserflow -Youtube(codebasic)](https://youtube.com/playlist?list=PLeo1K3hjS3uu7CxAacxVndI4bE_o3BDtO)
+- [Introduction to Deep Learning -MIT](http://introtodeeplearning.com/?fbclid=IwAR35rIygYlCn84DV7mlHvdvs4sMUm2D6RLYVwFpp2nT2t1Zj1GGy3QAWQvQ)
+- [Deep Learning Fundamentals -Youtube(deeplizard)](https://www.youtube.com/playlist?list=PLZbbT5o_s2xq7LwI2y8_QtvuXZedL6tQU)
+- [Deep Learning UC Berkeley STAT -Youtube(Alex Smola)](https://youtube.com/playlist?list=PLZSO_6-bSqHQHBCoGaObUljoXAyyqhpFW)
+- [Stanford Lecture - Convolutional Neural Networks for Visual Recognition -Youtube(Standford University of engineering)](https://youtube.com/playlist?list=PL3FW7Lu3i5JvHM8ljYj-zLfQRF3EO8sYv)
+- [CS480/680 -Youtube(Pascal Poupart)](https://youtube.com/playlist?list=PLdAoL1zKcqTW-uzoSVBNEecKHsnug_M0k)
+- [Dive into Deep Learning -Github(d21-ai)](https://github.com/d2l-ai/d2l-en?fbclid=IwAR0QN35b-NHHWq_zKISA1cbI063aRqqoKqR_0e3cpnT5h58GkcNbCIJs3iw)
+- [Introduction to Deep Learning -GFG(geeksforgeeks)](https://www.geeksforgeeks.org/introduction-deep-learning/)
+
+### PyTorch
+
+- [PyTorch -Youtube(Dr.Data Science)](https://www.youtube.com/watch?v=vVQs4h6HUvA&list=PLLeO8f6PhlKb_FAC7qxOBtxT9-8EPDAqk)
+- [PyTorch -Youtube(UC Berkeley) - Lec3 (The 5 parts)](https://m.youtube.com/watch?v=AOypIa_8RXg&list=PL_iWQOsE6TfXxKgI1GgyV1B_Xa0DxE5eH&index=11)
+- [Pytorch Tutorial -Youtube(Aladdin Persson)](https://www.youtube.com/watch?v=2S1dgHpqCdk&list=PLhhyoLH6IjfxeoooqP9rhU3HJIAVAJ3Vz)
+- [PyTorch Course (2022) - Youtube(Mr. p Solver)](https://www.youtube.com/watch?v=v43SlgBcZ5Y&list=PLkdGijFCNuVk9fO1IMfdV1Igob0FUHhkB)
+- [Deep Learning With Pytorch -GoogleDrive(Eli Stevens, Luca Antiga)](https://drive.google.com/file/d/1-KG_ufeg7zw2iLgG5RrJSFpyonLwulgF/view?usp=sharing)
+
+### Natural Language Processing (NLP)
```

Comment:
Again, NLP is not necessary in DS for a long time.

Reply:
- removed it